cypress/integration/mytools.js

```diff
@@ -211,14 +218,19 @@ describe('Dockstore my tools', function() {
 
   describe('manually register a SBG tool', function() {
     it("register tool", function() {
+      var toolObject = {"id": 40000, "author":null,"description":null,"labels":[],"users":[{"id":1,"username":"user_A","isAdmin":false,"name":"user_A"}],"email":null,"defaultVersion":null,"lastUpdated":1482334377743,"gitUrl":"git@github.com:testnamespace/testname.git","mode":"MANUAL_IMAGE_PATH","name":"testname","toolname":"","namespace":"testnamespace","registry":"AMAZON_ECR","lastBuild":null,"tags":[],"is_published":false,"last_modified":null,"default_dockerfile_path":"/Dockerfile","defaultCWLTestParameterFile":"/test.cwl.json", "defaultWDLTestParameterFile":"/test.wdl.json","default_cwl_path":"/Dockstore.cwl","default_wdl_path":"/Dockstore.wdl","tool_maintainer_email":"test@email.com","private_access":true,"path":"images.sbgenomics.com/testnamespace/testname","tool_path":"images.sbgenomics.com/testnamespace/testname", "custom_docker_registry_path": "images.sbgenomics.com"}
```

Comment:
this is near identical to the other response. just clone the previous and change the different fields